package.json

```diff
@@ -63,6 +63,7 @@
     "@semantic-release/changelog": "^5.0.1",
     "@semantic-release/exec": "^5.0.0",
     "@semantic-release/git": "^9.0.0",
+    "@types/stylelint": "^14.0.0",
```

Comment:
- [ ] Those types are out of date too - https://github.com/stylelint/stylelint/issues/2444